recipe/meta.yaml

```diff
@@ -26,11 +26,10 @@ requirements:
     - cmake  # [win]
     - make  # [not win]
     - wget  # [not win]
-    - pkgconfig
+    - pkg-config
   host:
     - libblas
     - liblapack
-    - metis
     - mumps-seq
     - scotch
```

Comment:
```suggestion
```

Reply:
oops, that's what I get for rebasing concurrent PRs. Thanks!